src/components/SigninModal/SigninModal.js

```diff
@@ -46,16 +49,12 @@ const SigninModal = ({ onOpenModal, onCloseModal, isLoading, onUserLogin }) => {
     return formIsValid;
   };
 
-  const handleSubmit = (formData) => {
-    formData.preventDefault();
-    debugger;
-    console.log(formData.target.elements[0].value);
+  const handleSubmit = (e) => {
+    e.preventDefault();
     if (validateForm()) {
+      setFormErrors({});
       // Submit form data and close modal
-      onUserLogin(
-        formData.target.elements[0].value,
-        formData.target.elements[1].value
-      );
+      onUserLogin(formData);
       onCloseModal();
```

Comment:
`   onCloseModal();`

You need to close popups only in blocks `then` after a **successful response** from the server so that when there is an error it would be opened (the inputs will not be cleared and the user could send the data again to the server) and if you change the button text you could see the change as well